### DISCUSSION TITLE
Choose different key than `spacebar` for modifying pan/zoom

### DISCUSSION THREAD

**Discussion by m4ttheweric** [reaviz/reagraph#191] [Q&A]
Is there a way to specify a different keyboard key for changing the pan/zoom behavior other than the spacebar?

The issue is that spacebar, if there's overflow on the page, will scroll to the bottom.

Would be cool to be able to use command/control/shift or something.

**Comment by amcdnl** [comment]
We could change it to prevent that default behavior.

**Reply by m4ttheweric** [reply]
A prop that allows a custom modifier or a string union of options seems like it would be a good solution.
